src/stores/room-list/previews/MessageEventPreview.ts

```diff
@@ -57,13 +55,24 @@ export class MessageEventPreview implements IPreview {
         }
 
         if (hasHtml) {
-            const sanitised = getHtmlText(body.replace(/<br\/?>/gi, "\n")); // replace line breaks before removing them
+            const sanitised = body.replace(/<br\/?>/gi, "\n"); // replace line breaks before removing them
             // run it through DOMParser to fixup encoded html entities
-            body = new DOMParser().parseFromString(sanitised, "text/html").documentElement.textContent;
+            const document = new DOMParser().parseFromString(sanitised, "text/html").documentElement;
+            document.querySelectorAll("[data-mx-spoiler]").forEach(spoiler => spoiler.textContent = "[spoiler]");
```

Comment:
Would it make sense to allow localization of the "[spoiler]" text?

Reply:
Makes sense to me!